src/test/java/seedu/revision/ui/StartQuizWindowTest.java

```diff
@@ -125,7 +125,9 @@ public void typeValidAnswer_endOfLevelReached_shouldShowNextLevelAlert(FxRobot r
         robot.write(ANSWER_VALID_MCQ);
         robot.type(KeyCode.ENTER);
         int numberOfAlerts = getNumberOfWindowsShown(robot, AlertDialog.NEXT_LEVEL_TITLE);
-        assertEquals(1, numberOfAlerts);
+        assertEquals(
```

Comment:
Does this pass checkstyle? Can check the formatting in the source code.

Reply:
Changed formatting